lib/utils/authentication.js

```diff
@@ -15,6 +15,7 @@ const _ = require('lodash');
 
 module.exports.register = function(server, options, next) {
     const redisClient = server.plugins['hapi-redis'].client;
+    const relations = Bluebird.promisifyAll(server.plugins.relations);
```

Comment:
Yeah, it would probably be good to promisify relations in a more 'global' place. How about as a `afterRegistration` function in _lib/utils/get-plugins.js_? `hapi-redis` uses an `afterRegistration` function for the same purpose in _lib/utils/get-plugins.js_, and you can use that as a guide.